tools/multicluster/go.mod

```diff
@@ -1,34 +1,36 @@
 module github.com/10gen/ops-manager-kubernetes/multi
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.4
 
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/spf13/cobra v1.6.1
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.4
 	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2
-	k8s.io/api v0.27.12
-	k8s.io/apimachinery v0.27.12
-	k8s.io/client-go v0.27.12
-	k8s.io/utils v0.0.0-20240310230437-4693a0247e57
+	k8s.io/api v0.28.11
```

Comment:
This should be incremented to [1.28.13](https://git.k8s.io/kubernetes/CHANGELOG/CHANGELOG-1.28.md#v12813)